src/ConfigUpdaterInterface.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\commerce;
+
+/**
+ * Performs configuration updates on behalf of an extension.
+ *
+ * Service to provide methods for extensions to import, revert, or delete
+ * configuration once they have been installed. Intended to be run from
+ * hook_post_update_NAME(), since this involves the modification of entities.
+ *
+ * @see hook_post_update_NAME()
+ */
+interface ConfigUpdaterInterface {
+
+  /**
+   * Imports configuration from extension storage to active storage.
+   *
+   * This action triggers a ConfigRevertInterface::IMPORT event if the
+   * configuration could be imported.
+   *
+   * @param string[] $config_names
+   *   An array of configuration names.
+   *
+   * @return \Drupal\commerce\ConfigUpdaterResultInterface
+   *   Object containing failed and succeeded configuration object update.
+   *   May also throw exceptions if there is a problem during saving the
```

Comment:
There's no such thing as "may also throw exceptions". Either the method throws exceptions, which are explicitly documented with @throws, or it doesn't (cause the exceptions are called by a method called by this method, for example), in which case no mention is needed.